config/dev.exs

```diff
@@ -65,3 +65,5 @@ config :phoenix, :stacktrace_depth, 20
 
 # Initialize plugs at runtime for faster development compilation
 config :phoenix, :plug_init_mode, :runtime
+
+config :tic_tac_toe_api, env: Mix.env
```

Comment:
You don't need to repeat this line in the three environment files. I think you can just put it in the `config.exs` file once

Reply:
😮